packages/pyright-internal/src/languageService/indentationUtils.ts

```diff
@@ -421,26 +418,364 @@ function _getLineEndingLength(parseResults: ParseResults, line: number) {
     return length;
 }
 
-function _getIndentationFromText(parseResults: ParseResults, line: number) {
+function _getIndentationFromText(
```

Comment:
For consistency with the rest of the code base, can you move private functions below exported functions within the module?

Reply:
sure.